docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1326,6 +1326,36 @@ Valid values are `-1` (unlimited), `integer` (in kilobytes), `nMB` (in megabytes
 
 Default: `1GB`
 
+## Catalog flags
+
+##### ysql_catalog_preload_additional_table_list
+
+Specifies the names of catalog tables (such as pg_operator, pg_proc, and pg_amop) to be preloaded by PostgreSQL backend processes. This flag reduces latency of first execution of a particular statement on a connection.
```

Comment:
This flag reduces latency of first execution of a particular statement on a connection. -> This flag reduces the first query execution latency on new connections.